### ISSUE TITLE
🪲 Bug Report - Extra parameter in policy_assignment_es_deny_resource_locations.tmpl.json policy assignment.

### ISSUE THREAD

**Issue by evgenyb** [Azure/ALZ-Bicep#585]
## Describe the bug

Extra parameter in `Deny-Resource-Locations` (policy_assignment_es_deny_resource_locations.tmpl.json) policy assignment. 

`Deny-Resource-Locations` uses [Allowed locations](https://www.azadvertizer.net/azpolicyadvertizer/e56962a6-4747-49cd-b67b-bf8b01975c4c.html) policy which only uses one parameter - `listOfAllowedLocations`, while `policy_assignment_es_deny_resource_locations.tmpl.json` file contains two parameters:

```json
...
"parameters": {
  "effect": {
    "value": "deny"
  },
  "listOfAllowedLocations": {
    "value": [
      "uksouth",
      "ukwest"
    ]
  }
}
...
```

## To Reproduce

1. Add new policy assignment into `alzDefaultPolicyAssignments.bicep` file (followed by [this documentation](https://github.com/Azure/ALZ-Bicep/wiki/AssigningPolicies#how-do-i-extend-the-alz-bicep-alz-default-policy-assignments-module))
2. Run `Deploy-ALZPolicyAssignments.ps1` script (or deploy it with `ALZ-Bicep-2-PolicyAssignments` pipeline)

You will get the following error message:

> The policy assignment 'Deny-Resource-Locations' has the parameter(s) 'effect' which are not defined in the policy definition 'e56962a6-4747-49cd-b67b-bf8b01975c4c'.

## Expected behaviour

No errors and Policies are assigned successfully

## Additional context

After removing `effect` parameter from `policy_assignment_es_deny_resource_locations.tmpl.json` file - all is good :)

**Comment by jtracey93** [comment]
Good spot @evgenyb, I have fixed this and will make it out in `v0.16.0` which will likely be today

**Comment by jtracey93** [comment]
Thanks @evgenyb,

Updated all of these :)
